.changeset/short-sheep-divide.md

```diff
@@ -0,0 +1,8 @@
+---
+"@fuel-ts/account": minor
+"@fuel-ts/interfaces": patch
+"@fuel-ts/contract": patch
+"@fuel-ts/program": patch
```

Comment:
@petertonysmith94 Shouldn't be `minor` for everyone here?  

The packages `contract`, and `program` are using the `account` package, and `interfaces` is being used by `program`?

Reply:
The rationale behind this was that the `@fuel-ts/account` is the only package with breaking changes.

Changed: b9058e0812bb123d3f1dd1be1bae4eadbaddd9e8